messages/rule-viewer.md

```diff
@@ -0,0 +1,12 @@
+# summary.found-no-rules
+
+Found 0 rules.
+
+# summary.found-rules
+
+Found %s rules:
```

Comment:
Does the messaging utilities from sf allow us to be %d instead of %s for number based inputs?

Reply:
I'm not sure. I'll check and report back.

Reply:
Yes it does. Changing that.

Reply:
@stephen-carter-at-sf 
To clarify, it _does_ allow you to specify `%d`, but instead of giving you a developer error or something if you pass in a non-numeric string, it'll just print "NaN". I'm not sure if that's a desirable alternative. Thoughts?

Reply:
Well - we shouldn't pass in non-strings which give us NaN. :-)
I simply like the readability of %d and I'm sure doc writers would appreaciate knowing that this whole is not filled with a string... but filled with a number.

Reply:
Fair enough. Will do.